### PR TITLE
[DRUP-536] Fix the title bar styles, improve formatted text on mobile.

### PR DESCRIPTION
**Pull request by jacine**
## Large
title-bar large

## Small
title-bar small

Edit: ^ I accidentally had "started" text selected w/my mouse, it's not actually that color. 😛

### CODE REVIEW

File: themes/custom/apigee_kickstart/assets/css/apigee-kickstart.style.css [hunk 1]
```diff
@@ -10191,7 +10196,7 @@ ul.social-links li a {
 }
 
 .title-bar__content {
-  border-color: #555 !important;
+  border-color: #a3a3a3 !important;
```
Comment (minnur) [inline themes/custom/apigee_kickstart/assets/css/apigee-kickstart.style.css:10199]: @jacine should this be a variable ?

Comment (minnur) [reply themes/custom/apigee_kickstart/assets/css/apigee-kickstart.style.css:10199]: nm, this is already a variable, looking at compiled file.

**Review by minnur — APPROVED** [review]
Looks good to me.

File: themes/custom/apigee_kickstart/assets/css/apigee-kickstart.style.css (hunk 1 above)
Comment (jacine) [reply themes/custom/apigee_kickstart/assets/css/apigee-kickstart.style.css:10199]: Thanks Minnur! :)

**Pull request merged**